packages/sanity/src/core/form/inputs/PortableText/object/BlockObject.tsx

```diff
@@ -151,7 +151,7 @@ export function BlockObject(props: BlockObjectProps) {
       __unstable_boundaryElement: boundaryElement || undefined,
       __unstable_referenceElement: memberItem?.elementRef?.current || undefined,
       children: input,
-      focused,
+      focused: memberItem?.node.focused,
```

Comment:
I'm not 100% sure about this one, the previous `focused` prop seems to come from the Portable Text editor and is based on selection. Would love to have your eyes on this @skogsmaskin.